cds_books/config.py

```diff
@@ -128,22 +128,31 @@ def _parse_env_bool(var_name, default=None):
 
 # Celery configuration
 # ====================
-BROKER_URL = 'amqp://guest:guest@localhost:5672/'
+BROKER_URL = "amqp://guest:guest@localhost:5672/"
 #: URL of message broker for Celery (default is RabbitMQ).
-CELERY_BROKER_URL = 'amqp://guest:guest@localhost:5672/'
+CELERY_BROKER_URL = "amqp://guest:guest@localhost:5672/"
 #: URL of backend for result storage (default is Redis).
-CELERY_RESULT_BACKEND = 'redis://localhost:6379/2'
+CELERY_RESULT_BACKEND = "redis://localhost:6379/2"
+#: Scheduled tasks configuration (aka cronjobs).
+CELERY_BEAT_SCHEDULE = {
+    **CELERY_BEAT_SCHEDULE,  # Parent config
```

Comment:
minor: Maybe rename it in `ILS_CELERY_BEAT_SCHEDULE` just to be clearer?

Reply:
This was intended, in order to emphasize that the field 'kind' is the same and has to come from the parent. As a matter of taste I prefer it the current way, but I can change it if needed.